samples/mentions.html

```diff
@@ -0,0 +1,271 @@
+						</textarea>
+					</div>
+					<button id="send" class="send">Send</button>
+				</div>
+
+			</section>
+		</section>
+
+		<footer class="sdk-footer">
+		</footer>
+
+		<script data-sample="1" >
+			var users = [
+				{ id: 1, avatar: 'm_1', fullname: 'Charles Flores' },
+				{ id: 2, avatar: 'm_2', fullname: 'Gerald Jackson' },
+				{ id: 3, avatar: 'm_3', fullname: 'Wayne Reed' },
+				{ id: 4, avatar: 'm_4', fullname: 'Louis Garcia' },
+				{ id: 5, avatar: 'm_5', fullname: 'Roy Wilson' },
+				{ id: 6, avatar: 'm_6', fullname: 'Matthew Nelson' },
+				{ id: 7, avatar: 'm_7', fullname: 'Randy Williams' },
+				{ id: 8, avatar: 'm_8', fullname: 'Albert Johnson' },
+				{ id: 9, avatar: 'm_9', fullname: 'Steve Roberts' },
+				{ id: 10, avatar: 'm_10', fullname: 'Kevin Evans' },
+
+				{ id: 11, avatar: 'w_1', fullname: 'Mildred Wilson' },
+				{ id: 12, avatar: 'w_2', fullname: 'Melissa Nelson' },
+				{ id: 13, avatar: 'w_3', fullname: 'Kathleen Allen' },
+				{ id: 14, avatar: 'w_4', fullname: 'Mary Young' },
+				{ id: 15, avatar: 'w_5', fullname: 'Ashley Rogers' },
+				{ id: 16, avatar: 'w_6', fullname: 'Debra Griffin' },
+				{ id: 17, avatar: 'w_7', fullname: 'Denise Williams' },
+				{ id: 18, avatar: 'w_8', fullname: 'Amy James' },
+				{ id: 19, avatar: 'w_9', fullname: 'Ruby Anderson' },
+				{ id: 20, avatar: 'w_10', fullname: 'Wanda Lee' }
+			],
+
+			buttonEl = document.getElementById( 'send' );
+
+		// Lets create some fake usernames from fullnames.
+		CKEDITOR.tools.array.forEach( users, function( user ) {
+			var username = user.fullname.split( ' ' );
+			username[ 0 ] = username[ 0 ].substring( 0, 1 );
+			user.username = username.join( '' ).toLowerCase();
+		} );
+
+		CKEDITOR.replace( 'editor1', {
+			extraPlugins: 'mentions,emoji',
+			removeDialogTabs: 'link:advanced',
+			removePlugins: 'elementspath',
+			height: 150,
+			resize_enabled: false,
+			toolbar: [
+				{ name: 'document', items: [ 'Undo', 'Redo' ] },
+				{ name: 'basicstyles', items: [ 'Bold', 'Italic', 'Strike', '-', 'RemoveFormat' ] },
+				{ name: 'links', items: [ 'Link', 'Unlink' ] }
+			],
+			mentions: [
+				{
+					feed: dataFeed,
+					itemTemplate:
+					'<li data-id="{id}"><img class="photo" src="assets/mentions/img/{avatar}.jpg" />' +
+						'<strong class="username">{username}</strong>' +
+						'<span class="fullname">{fullname}</span>' +
+					'</li>',
+					outputTemplate: '<a href="mailto:{username}@example.com">@{username}</a><span>&nbsp;</span>',
+					minChars: 0
+				}
+			],
+			on: {
+				instanceReady: function( evt ) {
+					buttonEl.addEventListener( 'click', function() {
+						evt.editor.showNotification( 'Your message has been send! However this is only example chat and we cannot make your post public.', 'success', 8000 );
+					} );
+				}
+			}
+		} );
+
+		function dataFeed( opts, callback ) {
+			 var data = users.filter( function( item ) {
```

Comment:
I don't find comparing `item.fullname.replace( ' ', '' )` with `opts.query` to be a necessity here. Let it be as simple as possible, so just compare based on user name.